mttl/models/expert_config.py

```diff
@@ -63,7 +63,7 @@ def _set_defaults(self):
         self.eval_mmlu_few_shot = True  # use few-shot for mmlu, default
         self.eval_mmlu_flag = False  # eval mmlu performance during training
         self.eval_rouge_flag = False  # eval rouge during training
-        self.pipeline_eval_tasks = "all"
+        self.pipeline_eval_tasks = None
```

Comment:
Just double checking.. Is the idea of this change to have eval disabled by default, or to use all if `pipeline_eval_tasks = None`

Reply:
I think setting "all" as default is better. "None" causes "no task find" error.

Reply:
is to don't eval on tasks by default, does this work?